app/views/diggers/_from.html.erb

```diff
@@ -0,0 +1,11 @@
+<div class="container">
```

Comment:
what file is it? _from? do you want to create a reusable component? than go in a new folder shared and call it form

Reply:
i delete this one